app/liquid_tags/organization_tag.rb

```diff
@@ -22,7 +23,15 @@ def render(_context)
   end
 
   def parse_slug_to_organization(organization)
-    organization = Organization.find_by(slug: organization)
+    if organization.include?(URL.url.to_s)
```

Comment:
clarity: this check says "did this link point to the site?" - is there a better way to do this than calling URL.url? it took a second to determine what this said, I'm just looking for something that says to the reader "this is the site's url" in an unambiguous way.

Reply:
nitpick: https://github.com/forem/forem/blob/main/app/lib/url.rb#L19 we already cast this to string before returning (URL.url returns the string representation of the site) - the `to_s` call is unneeded.

Reply:
should this say `organization.starts_with?(URL.url)`

Reply:
I guess `starts_with` is more specific 🤷🏾‍♀️  Ok I will change